packages/components/checkbox/index.js

```diff
@@ -71,47 +77,49 @@ export const Checkbox = ({
       ]),
     },
     label: {
-      ml: isReversed ? '0' : '2',
-      mr: isReversed ? '2' : '0',
-      fontFamily: 'body',
-      fontSize: 'checkbox',
-      fontWeight: 'medium',
-      letterSpacing: 'checkbox',
-      color: `${
-        isDisabled ? 'font.checkbox.disabled' : 'font.checkbox.default'
-      }`,
+      ml: isReversed ? '0' : label.normal.margin,
+      mr: isReversed ? label.normal.margin : '0',
+      fontFamily: label.normal.fontFamily,
+      fontSize: label.normal.fontSize,
+      fontWeight: label.normal.fontWeight,
+      letterSpacing: label.normal.letterSpacing,
+      color: `${isDisabled ? label.disabled.color : label.normal.color}`,
       ...getAllowedProps(override.label, [...color, ...space, ...typography]),
     },
   };
 
   return (
-    <Box as="label" {...Override.wrapper}>
-      {/* This is the sibling input, it's visually hidden */}
-      <VisuallyHidden
-        as="input"
-        type="checkbox"
-        id={id}
-        name={name}
-        value={value}
-        checked={isChecked}
-        disabled={isDisabled}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        onChange={onChange}
-        defaultChecked={defaultIsChecked}
-        {...Override.visuallyHidden}
-      />
+    <ThemeProvider theme={theme}>
+      <Box p="2">
+        <Box as="label" {...Override.wrapper}>
+          {/* This is the sibling input, it's visually hidden */}
+          <VisuallyHidden
+            as="input"
+            type="checkbox"
+            id={id}
+            name={name}
+            value={value}
+            checked={isChecked}
+            disabled={isDisabled}
+            onBlur={onBlur}
+            onFocus={onFocus}
+            onChange={onChange}
+            defaultChecked={defaultIsChecked}
+            {...Override.visuallyHidden}
```

Comment:
We don't have overrides for visually hidden

Reply:
Fixed it. I did not notice it. Thanks 😄